external/storm-elasticsearch/README.md

```diff
@@ -0,0 +1,66 @@
+# Storm ElasticSearch Bolt & Trident State
```

Comment:
This search engine was ElasticSearch as you wrote, but is Elasticsearch now.
In short, the 2nd "s" is a small "s".
https://www.elastic.co/products/elasticsearch